cfme/tests/infrastructure/instance.py

```diff
@@ -0,0 +1,648 @@
+""" A model of Instances page in CFME."""
```

Comment:
No idea what's going on here. This seems like a copy-paste of `cloud/instance.py` into `infrastructure/instance.py`.